composer.json

```diff
@@ -7,21 +7,16 @@
         "php": ">=8.1",
         "nette/utils": "^3.2.9 || ^4.0",
         "webmozart/assert": "^1.11",
-        "phpstan/phpstan": "^1.11",
+        "phpstan/phpstan": "^2.0",
         "symplify/rule-doc-generator-contracts": "^11.2"
     },
     "require-dev": {
         "nikic/php-parser": "^4.19",
-        "symplify/phpstan-extensions": "^11.4",
         "symplify/rule-doc-generator": "^12.2",
         "phpunit/phpunit": "^10.5",
         "symfony/framework-bundle": "6.1.*",
-        "rector/rector": "^1.2.3",
```

Comment:
could you use 

```php
        "rector/rector": "dev-main as 1.2.10",
```

to test rector for CI?

Reply:
I applied it to https://github.com/symplify/phpstan-rules/pull/144/commits/f83aafa22ee855df2305aab9be427dec162c278e